### PR TITLE
Fix driver import errors

### PR DESCRIPTION
**Pull request by John2202W**
Now catch the pyvisa exception when a driver cannot be opened and raise a more informative error message on top.
- Previous: 
> pyvisa.errors.VisaIOError: VI_ERROR_RSRC_NFOUND (-1073807343): Insufficient location information or the requested device or resource is not present in the system.
-  New: 
> fixate.drivers.InstrumentOpenError: Unable to open DSO: USB0::0x2A8D::0x1797::CN58166436::INSTR

Fixed circular import in pps driver.
Remove a missed old pyvisa import from pytests (don't believe it is run atm anyway)

### CODE REVIEW

**Comment by John2202W** [comment]
@jcollins1983 - if you are doing some fixate reviews (this may impact changes to the DMM __init__ in PR #162 aswell) - would be good to get this one in the pipeline.
Giving info on the failed driver would help production out a whole heap instead of pyvisa error xxxxx, they can at least try re-plugging or cycling power on the relevant instrument.

**Review by jcollins1983 — COMMENTED** [review]
There's a couple of comments that could do with an update as indicated. There are a few "dittos" because of the copy/paste nature of these drivers. I feel like there's a better way, anyway, not the point of this review.
Once you've fixed up those comments I'll be happy to merge.

File: src/fixate/core/config_util.py [hunk 1]
```diff
@@ -366,6 +366,7 @@ def visa_id_query(visa_resource_name):
 
     # At least one instrument (Siglent SPD3303X power supply) only responds
     # when the line termination is set to \n
+    # Could the previous un-terminated query be causing issues with pps connectivity?
```
[the remark this answers is not in the document]
Comment (John2202W) [reply src/fixate/core/config_util.py]: Now deleted - notes about another issue that I'll stash elsewhere

**Review by jcollins1983 — APPROVED** [review]

File: src/fixate/drivers/dmm/__init__.py [hunk 2]
```diff
@@ -8,19 +8,26 @@
...
 from fixate.config import find_instrument_by_id
+from fixate.drivers import InstrumentNotFoundError, InstrumentOpenError
+from fixate.drivers.dmm.fluke_8846a import Fluke8846A
+from fixate.drivers.dmm.helper import DMM
 
 
 def open() -> DMM:
     instrument = find_instrument_by_id(Fluke8846A.REGEX_ID)
     if instrument is not None:
         # we've found a connected instrument so open and return it
```
[the remark this answers is not in the document]
Comment (John2202W) [reply src/fixate/drivers/dmm/__init__.py]: Done now - check if that's clearer

**Comment by John2202W** [comment]
> There's a couple of comments that could do with an update as indicated. There are a few "dittos" because of the copy/paste nature of these drivers. I feel like there's a better way, anyway, not the point of this review. Once you've fixed up those comments I'll be happy to merge.

I think a generic "visa instrument" helper would be super useful - have discussed this in the past that specific drivers can inherit and prevent all the duplicated visa related setups/controls.

**Pull request merged**